spec/view_helpers_spec.rb

```diff
@@ -60,4 +60,10 @@ def helper_output(honeypot = nil, scope = nil, options = {})
       expect(invisible_captcha(visual_honeypots: false)).to eq(helper_output(nil, nil, visual_honeypots: false))
     end
   end
-end
\ No newline at end of file
+
+  it 'should set spam timestamp' do
+    InvisibleCaptcha.honeypots = [:foo_id]
+    expect(invisible_captcha).to eq(helper_output)
```

Comment:
I think this `expect` isn't necessary, just a call to the method `invisible_captcha`.